.github/workflows/python-package.yml

```diff
@@ -5,7 +5,7 @@ name: Python package
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "01_tests" ]
```

Comment:
1) We use `master` branch instead of `main`.
2) I don't think you need to run tests for any branches except `master`. Tests can be run locally with `ci/test.sh` script.